### ISSUE TITLE
Grab a single item from a stack?

### ISSUE THREAD

**Issue by Kiefy** [YaLTeR/MouseTweaks#42]
Similar to the r-click to split a stack, is there some way to take just 1 item?  having it stuck to your mouse and ready to drop wherever you please.

I'm sure I remember that being a thing at some point, but I'm not sure if it was Mouse Tweaks, Inv Tweaks or something else.

If it isn't possible. might I offer it as a feature suggestion? :)

**Comment by YaLTeR** [comment]
Yeah, this was never in Mouse Tweaks. How would you trigger such an action?
